studies/templates/studies/lab_create.html

```diff
@@ -16,7 +16,7 @@
         {% csrf_token %}
         {% bootstrap_form form %}
         {% form_buttons %}
-            {% bootstrap_button "Submit" %}
-        {% endform_buttons %}
-    </form>
+        {% bootstrap_button "Submit" %}
+    {% endform_buttons %}
```

Comment:
@okaycj one other minor thing I forgot to mention: I noticed that our new linting tool isn't indenting some of these template tags properly. E.g. `form_buttons`/`endform_buttons` isn't recognized as the start/end of an indented code block here (and in several other files). Is there any way to fix this? (Obviously this shouldn't hold up merging this PR - if it's not an easy fix then I'll put it in an issue).

Reply:
Yeah, I noticed this as well.  It's not great and I hope they fix it in the future.  If it's a real problem, I have no problem with submitting an issue.